src/packet/ipv6.rs

```diff
@@ -22,7 +22,7 @@ pub struct Ipv6 {
     flow_label: u20be,
     payload_length: u16be,
     #[construct_with(u8)]
-    next_header: IpNextHeaderProtocol,
+    next_level_protocol: IpNextHeaderProtocol,
```

Comment:
Why was this field renamed? It was intentionally named that way to match the specification.